.travis.yml

```diff
@@ -1,16 +1,14 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.0
-  - 2.4.4
-  - 2.5.0
-  - 2.5.1
-  - 2.5.2
-  - 2.5.3
+  - 2.5.5
```

Comment:
What is the reasoning behind continuing to support `2.5.5` when we are moving to `2.6.x`? Is `2.6.x` not available in some envs?

Reply:
with gems you should support older versions too, unless you are desupporting a ruby version entirely. When built we supported 2.4+ but since it is quite old and CCCD does not use and it means the codebase has to cater for it unnecessarily (the spec helpers needed nested `begin;rescue;end;` blocks for 2.4 support for example) I have decided to remove it. In fact it would work with it, just not the test suite

Reply:
Also, we are supporting 2.5+ but only testing against latest 2.5+ (2.5.5) - see the gemspec